bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -262,6 +267,291 @@ def predict(self):
                 if class_id == 1:
                     self._goalpost_candidates.append(c)
 
+class YoloHandlerNCS2(YoloHandler):
+    """
+    This Code is based on a code example from the Intel documentation under following licensing:
```

Comment:
```suggestion
    The following code is based on a code example from the Intel documentation under following licensing:
```
I suggest also mentioning the link to the doc.